### PR TITLE
RollDice

### LINKED ISSUE #106: Dice Rolling Simulator using Python-random
* Aim
Dice Rolling Simulator using Python-random

* Details
In this Script we will create a classic rolling dice simulator with the help of basic Python knowledge. Here we will be using the random module since we randomize the dice simulator for random outputs.

### PR DESCRIPTION
**Pull request by devmalik7**
Fixes #106 

### Checklist:
- [x] My branch is up-to-date with upstream/master branch.

### CODE REVIEW

**Comment by Tejas1510** [comment]
The Readme File is missing Input and Output Images

File: Java/RollDice/RollDice.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import java.util.Random;
+
+public class RollDice {
+
+	private static Scanner userchoice;
+	
+	public static void main (String[] args)
+	{
+		//Start option for dice roll//
+		System.out.println("Do you want to roll the dice? (Y/N)");
```
Comment (atarax665) [inline Java/RollDice/RollDice.java:11]:
```suggestion
		System.out.println("Do you want to roll the dice? (y/n)");
```

File: Java/RollDice/RollDice.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+			//If user chooses to roll the dice//
+			else if (choice.equals("y"))
+			{
+				int faces;
+				
+				Random generator = new Random();
+				faces = generator.nextInt(6) + 1;
+				
+				System.out.println("You rolled a " + faces + ".");
+				System.out.println("Roll again?");
```
Comment (atarax665) [inline Java/RollDice/RollDice.java:38]:
```suggestion
				System.out.println("Roll again? (y/n)");
```

File: Java/RollDice/RollDice.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+			else if (choice.equals("n"))
+			{
+				System.out.println("Thank you for rolling!");
+				break;
+			}
+		}	
+		
+	}
+	
+}
```
Comment (atarax665) [inline Java/RollDice/RollDice.java:54]: Add a line break at the EOF.

File: Java/RollDice/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+## Introduction
+```
+It is the program that will help you to Roll the Dice.
+```
+
+## Requirements
+```
+JDK
+JRE
```
Comment (atarax665) [inline Java/RollDice/README.md:11]: As the program runs in terminal so user can use any IDE of his choice to run this program. Only requirement should be Java IDE.

File: Java/RollDice/README.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# RollDice  
+
+## Introduction
+```
+It is the program that will help you to Roll the Dice.
```
Comment (atarax665) [inline Java/RollDice/README.md:5]: Please elaborate the introduction to how this program used ```Random ()``` function to achieve the task.

File: Java/RollDice/README.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+JDK
+JRE
+```
+
+
+## How to use the code
+```
+1.Download the given code.
+2.Open the project .
+3.Run the .JAVA file.
```
Comment (atarax665) [inline Java/RollDice/README.md:19]:
```suggestion
3.Run the RollDice.java file.
```

File: Java/RollDice/README.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+It is the program that will help you to Roll the Dice.
+```
+
+## Requirements
+```
+JDK
+JRE
+```
+
+
```
Comment (atarax665) [inline Java/RollDice/README.md:14]: Remove this extra blank line.

File: Java/RollDice/README.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+## How to use the code
+```
+1.Download the given code.
+2.Open the project .
+3.Run the .JAVA file.
+```
+
```
Comment (atarax665) [inline Java/RollDice/README.md:21]: Remove this extra blank line.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**